src/macros.jl

```diff
@@ -188,6 +188,15 @@ function parse_constraint(_error::Function, sense::Symbol, lhs, rhs)
     (sense, vectorized) = _check_vectorized(sense)
     vectorized, parse_one_operator_constraint(_error, vectorized, Val(sense), lhs, rhs)...
 end
+function parse_constraint(_error::Function, ::Val{:call}, args...)
```

Comment:
Sorry, my recommendation doesn't work. Why not replace
```julia
function parse_constraint(_error::Function, args...)
    _unknown_constraint_expr(_error)
end
```
by
```julia
function parse_constraint(_error::Function, args...)
    if args[1] isa Symbol
        (sense, vectorized) = _check_vectorized(args[1])
        vectorized, parse_one_operator_constraint(_error, vectorized, Val(sense), args[2:end]...)...
    else
        _unknown_constraint_expr(_error)
    end
end
```
and then in the tests, do
```julia
function JuMP.parse_one_operator_constraint(_error::Function, ::Val{:f}, x)
    return :(), :(build_constraint($_error, $(esc(x)), $(esc(CustomType()))))
end
```

Reply:
At least, this solution allows the test to pass, so I guess this is enough, thanks for your continued support :)!